docs/unraid-os/system-administration/maintain-and-update/changing-the-flash-device.md

```
@@ -0,0 +1,254 @@
+
+:::important Trial keys and device changes
+
+If you transfer a trial configuration to a new flash device, you will not be able to start the %%array|array%% until you purchase a valid registration key. Trial keys only work on the original device.
+:::
+
+### Replacement methods
+
+<Tabs>
+<TabItem value="creator" label="Using the USB flash creator" default>
+
+*Recommended for most users*
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)